packages/hash/frontend/src/blocks/page/BlockContextMenu/BlockContextMenu.tsx

```diff
@@ -34,27 +33,17 @@ import { BlockListMenuContent } from "./BlockListMenuContent";
 
 type BlockContextMenuProps = {
   blockEntity: BlockEntity | null;
-  blockSuggesterProps: BlockSuggesterProps;
```

Comment:
We don't need to drill this through props, the `BlockListMenuContent` can get the single function in it itself